### PR TITLE
Bug 1203507 - [Spanish] AMO Page title larger than screen width

### PR DESCRIPTION
**Pull request by tsl143**

### CODE REVIEW

**Comment by tsl143** [comment]
after
before

File: static/css/mobile/header.less [hunk 1]
```diff
@@ -181,9 +181,14 @@ header #auth-nav li {
...
     a {
         color: #000;
+        display: block;
         font: 36px/32px @serif-stack;
+        overflow: hidden;
         text-decoration: none;
+        text-overflow: ellipsis;
         text-transform: uppercase;
+        white-space: nowrap;    
+        width: 240px;
```
Comment (magopian) [inline static/css/mobile/header.less]: Is there a way to have that also work on other resolutions? ping @muffinresearch @tofumatt the CSS gurus ;)

**Comment by muffinresearch** [comment]
Hi @tsl143 As mentioned in the bug we already look to have a helper class for this: see https://github.com/mozilla/olympia/blob/d8aef0b88a5fd20d4413a37807e9ae79c7602fa8/static/css/impala/lib.less#L135 was there a reason to not use that - it will save on duplication.

File: static/css/mobile/header.less (hunk 1 above)
Comment (muffinresearch) [inline static/css/mobile/header.less]: Yep - I've noted the class to use in the other comment - but sounds like a plan to make that a goal that this should truncate long strings in both mobile and desktop.

**Comment by tsl143** [comment]
Hi @muffinresearch I tried using just the class, but it seems we need to give width else the ellipsis effect doesnt show up.

**Comment by muffinresearch** [comment]
@tsl143 that's fine - you should use the class and just add anything else needed where that's not enough.

**Comment by tsl143** [comment]
Hi @muffinresearch , I added ellipsis class to anchor and removed all repetitive definations from header.less. But on browser the ellipsis class is not defined ie., its defined in lib.less but not reflecting on the browser. Am I missing something again?

**Comment by muffinresearch** [comment]
Sorry my bad looks like this is a mixin. You should find it will work if you add `.ellipsis();`  in the css as opposed to adding an actual class to the markup.

**Comment by muffinresearch** [comment]
@tsl143 this is going to need rebasing against master. If you have a moment to do this and update to use the mixin, let me know when it's done and we can look to get this merged.

**Comment by tsl143** [comment]
@muffinresearch I am not sure if I understood what you mean by mixin? <sorry for being novice>

**Comment by tsl143** [comment]
@muffinresearch I got the mixin thing and updated the same :)

**Comment by muffinresearch** [comment]
@tsl143 ok it's close now. I think it's better to use 100% here for the width otherwise you're going to always cause truncation even if there's more width available.

E.g: as it stands with width fixed at 240px:


complementos_para_firefox

complementos_para_firefox

And with 100%:

complementos_para_firefox





complementos_para_firefox

**Comment by muffinresearch** [comment]
Actually scratch that, you can simply remove the width completely and it works correctly thanks to the `display: block`.

**Comment by muffinresearch** [comment]
Once you've removed the width can you squash the commits [1] and I'll get this merged.

[1] http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by tsl143** [comment]
@muffinresearch made the required changes, thanks for being patient :)

**Comment by muffinresearch** [comment]
@tsl143 excellent thanks. Merged!

**Pull request merged**